docker-compose.yml

```
@@ -0,0 +1,5 @@
+services:
+  graph:
+    image: tinkerpop/gremlin-server:3.7
+    ports:
+      - 8182:8182
```

```suggestion
      - 8183:8182
```
different internal - external port

---

That is usually done for reserved/assigned ports. In the case of port 8182 it is not assigned. If it ever conflicts with someone's local setup we can set this as an env var like
```yaml
ports:
  - ${GREMLEX_SERVER_PORT}:8182
```
and use an `.env` file like
```sh
GREMLEX_SERVER_PORT=8182
```